src/Console/Command.php

```diff
@@ -104,6 +105,23 @@ public function getName(): string
         return $this->name;
     }
 
+    /**
+     * Get the command name.
+     *
+     * @return string
+     */
+    public static function defaultName(): string
+    {
+        $pos = strrpos(static::class, '\\');
+        $name = substr(static::class, $pos + 1, -7);
+
+        return str_replace(
+            '_',
+            ' ',
+            Inflector::underscore($name)
```

Comment:
@ADmad we should use underscore name for user's commands 
your commit does not show this

Reply:
Hmm.. Why should user command not follow the same convention as core commands?

Reply:
I dont know 
https://book.cakephp.org/3.0/en/console-and-shells/commands.html#testing-commands ( search the `user_table` )
in document and cake 36./ 3.7 it is underscore by default

Reply:
Boo.. Okay, I'll update.

Reply:
in cake 3.x file names point to underscore (shell + command ) and subcommand to `space` 
we are facing with 2 concept :)

Reply:
Damn BC :rage: